code/modules/html_interface/voting/voting.js

```diff
@@ -35,7 +35,7 @@ function update_mode(newMode, newQuestion, newTimeleft, vrestart, vmode, vmap, v
 	allow_mode = parseInt(vmode) || 0;
 	allow_restart = parseInt(vrestart) || 0;
 	toggle_map = parseInt(vmap) || 0;
-	toggle_vote_method = parseInt(vmethod) || 0;
```

Comment:
why are you making this change?

Reply:
Defaulted to Null in the interface, which was not intended. This fixes it